recipes/python-funcsigs/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
+
+package:
+  name: python-{{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 7e583285b1fb8a76305d6d68f4ccc14e
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - funcsigs
```

Comment:
I suppose we could jinja template this, but we are not guaranteed that the package name will be what we use to import. Still it may work often enough to be a time-saver.